sdk/loadtesting/azure-developer-loadtesting/azure/developer/loadtesting/_generated/operations/_patch.py

```diff
@@ -178,7 +149,7 @@ async def begin_test_run(
         old_test_run_id: Optional[str] = None,
         content_type: str = "application/merge-patch+json",
         **kwargs: Any
-    ) -> LoadTestingLROPoller[JSON]:
+    ) -> LROPoller[JSON]:
```

Comment:
Same comment as for async, if we delete the two generated overloads these overloads in the `_patch.py` file wont conflict with them.